rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -811,8 +1049,21 @@ void CFontTexture::LoadWantedGlyphs(const std::vector<char32_t>& wanted)
 		if ((atlasUpdate.xsize != wantedTexWidth) || (atlasUpdate.ysize != wantedTexHeight))
 			atlasUpdate = atlasUpdate.CanvasResize(wantedTexWidth, wantedTexHeight, false);
 
-		if (atlasUpdateShadow.Empty())
-			atlasUpdateShadow.Alloc(wantedTexWidth, wantedTexHeight, 1);
+		if (needsColor && !isColor) {
+			CBitmap newAtlas = {};
+			newAtlas.Alloc(wantedTexWidth, wantedTexHeight, needsColor ? 4 : 1);
```

Comment:
Perhaps there's some sort of existent BGRA and GRAY constants for the 4 and 1? But I guess it's pretty obvious anyway so if you don't know off the top of your head then probably don't waste time searching.

Reply:
yeah don't know tbh, looked a bit and didn't find anything, anyways that's supposed to be the number of channels so I think a number is ok. for example at the constructor: `CBitmap(const uint8_t* data, int xsize, int ysize, int channels = 4, uint32_t reqDataType = 0);`